app/src/app/services/database-bots.service.ts

```diff
@@ -46,6 +48,19 @@ export class DatabaseBotsService {
     
     var postRetrun = this.httpClient.post<any>(this.SERVER_URL, this.formData(backendBot), httpOptions).subscribe(res =>{
       console.log(res)
+      // switch (res.type) {
```

Comment:
Jesli to juz kod po refaktorze, czy ten fragment kodu jest potrzebny? Jak jest, to dobrze by było dodać jakiś komentarz dlaczego :)